astrix-gs/src/main/java/com/avanza/astrix/gs/remoting/GsRemotingModule.java

```diff
@@ -50,4 +49,17 @@ public void prepare(ModuleContext moduleContext) {
 		moduleContext.export(ServiceComponent.class);
 	}
 
+	private void ensureGigaSpaceJavaCompatibility() {
```

Comment:
Note to reviewer: This is a new addition that is not present in the `gs14` branch. It should also be removed once the default gs version is bumped.